tools/devctr/Dockerfile

```diff
@@ -73,14 +73,13 @@ RUN apt-get update \
         # needed for integration tests
         net-tools iproute2 iperf3 socat fdisk \
         numactl \
+        iptables \
```

Comment:
why we need iptables now?

Reply:
@pb8o has more context here. Generally, whenever someone builds a new docker container, we usually perform other changes that other team members will need in the near future, so avoid having to repeatedly build docker containers.

Reply:
ok understood, so it is unrelated to this PR, thanks (maybe next time we can call out in the description of the PR), thanks.

Reply:
Added to PR description

Reply:
thanks